src/ensembl/src/content/app/species/state/sidebar/speciesSidebarSelectors.ts

```diff
@@ -17,8 +17,16 @@
 import { getActiveGenomeId } from 'src/content/app/species/state/general/speciesGeneralSelectors';
 import { RootState } from 'src/store';
 
-export const isSidebarOpen = (state: RootState) =>
-  state.speciesPage.sidebar.isOpen;
+export const isSidebarOpen = (state: RootState) => {
+  const activeGenomeId = getActiveGenomeId(state);
+
+  if (!activeGenomeId) {
+    return false;
+  }
+  return (
+    state.speciesPage.sidebar.species[activeGenomeId]?.isSidebarOpen ?? true
+  );
```

Comment:
![image](https://user-images.githubusercontent.com/6834224/108068167-a3f3db80-7059-11eb-9b06-c733321e0ed0.png)

shouldn't the default return values be the same?

Reply:
I thought this would mean that we keep the sidebar closed when there is no `activeGenomeId` and open it by default when we have `activeGenomeId`.